### ISSUE TITLE
breaks with yarn pnp

### ISSUE THREAD

**Issue by not-ivy** [ljharb/object.assign#86]
in version 4.1.6, this error occurs with yarn pnp:

```
Error: object.assign tried to access es-object-atoms, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Required package: es-object-atoms
Required by: object.assign@npm:4.1.6 (via /home/nora/.yarn/berry/cache/object.assign-npm-4.1.6-e03ad96fd6-10c0.zip/node_modules/object.assign/)
```

you can fix this by declaring the package as a dependency in `package.json` (just install it)

**Comment by not-ivy** [comment]
workaround by adding this in `.yarnrc.yml`

```yml
packageExtensions:
  object.assign@4.1.6:
    dependencies:
      es-object-atoms: ^1.0.0
```

**Comment by ljharb** [comment]
Thanks, I'll fix this in v4.1.7 (although this is an inherent flaw in PnP; it Just Works in a properly installed node_modules directory)

**Comment by ljharb** [comment]
v4.1.7 is published.
